packages/flutter/lib/src/material/page.dart

```diff
@@ -64,9 +124,6 @@ class MaterialPageRoute<T> extends PageRoute<T> {
   Color get barrierColor => null;
 
   @override
-  bool canTransitionFrom(TransitionRoute<dynamic> nextRoute) => false;
```

Comment:
Is there a way to query the platform without access to the BuildContext or Theme?